app/templates/components/lazy-image.hbs

```diff
@@ -1,26 +1,31 @@
 {{#if template}}
-  <div class="lazy-image-placeholder">{{yield}}</div>
+  {{#if errorThrown}}
```

Comment:
In chrome errorThrown is true by default. Maybe `lazyUrl: "//:0",` is causing the error...